specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2022-11-15-preview/managedCassandra.json

```diff
@@ -774,657 +693,1717 @@
+      "description": "Data center name in a managed Cassandra cluster.",
+      "minLength": 1,
+      "maxLength": 100,
+      "pattern": "^[a-zA-Z0-9]+(-[a-zA-Z0-9]+)*$"
+    },
+    "backupNameParameter": {
+      "name": "backupId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Id of a restorable backup of a Cassandra cluster.",
+      "minLength": 1,
+      "maxLength": 15,
+      "pattern": "^[0-9]+$"
+    },
+    "repairRunId": {
+      "name": "repairRunId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Id of repair run created to repair Cassandra cluster."
+    },
+    "ownerName": {
+      "name": "ownerName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Owner Name that was used to create the Repair Run."
+    },
+    "intensityValue": {
+      "name": "intensityValue",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Floating point value representing intensity of the repair."
+    },
+    "segmentId": {
+      "name": "segmentId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "description": "Id of a segment of a repair run."
+    }
+  },
+  "definitions": {
+    "ListClusters": {
+      "description": "List of managed Cassandra clusters.",
+      "readOnly": true,
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for the array of clusters.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClusterResource"
+          }
+        }
+      }
+    },
+    "ManagedCassandraARMResourceProperties": {
+      "type": "object",
+      "description": "The core properties of ARM resources.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The unique resource identifier of the ARM resource."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The name of the ARM resource."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "tags": {
+          "$ref": "cosmos-db.json#/definitions/Tags"
+        },
+        "identity": {
+          "$ref": "#/definitions/ManagedCassandraManagedServiceIdentity"
+        }
+      },
+      "x-ms-azure-resource": true
+    },
+    "ManagedCassandraManagedServiceIdentity": {
```

Comment:
>ManagedCassandraManagedServiceIdentity [](http://example.com/codeflow?start=5&length=38)

Consider replacing with references to the managed identity common-types in
[https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/common-types/resource-management/v5/managedidentity.json)